### ISSUE TITLE
mtime is not correct after untar

### ISSUE THREAD

**Issue by davies** [moosefs/moosefs#52]
It's expected that untar will preserve the stats of a file (for example, mtime), but in MooseFS, the mtime of files from untar are not preserved.

For example, untar  a file into local disk:
```
-rw-r--r-- 1 ubuntu ubuntu 997 Apr 19 16:07 README.txt
```

But in MooseFS:
```
-rw-r--r--     1 ubuntu ubuntu     997 Apr 25 06:07 README.txt
```

strace showed that tar will issue `utimensat` before `close`
```
openat(AT_FDCWD, "profiles/README.txt", O_WRONLY|O_CREAT|O_EXCL|O_NOCTTY|O_NONBLOCK|O_CLOEXEC, 0644) = 4
write(4, "Installation profiles define add"..., 997) = 997
fstat(4, {st_mode=S_IFREG|0644, st_size=997, ...}) = 0
utimensat(4, NULL, [{1493100371, 505698059}, {1492618042, 0}], 0) = 0
close(4)                                = 0
```
But in MooseFS, the mtime will be updated again in `close()`.

Should we tracking the mtime in client (during write()), and update it during `setattr`. When we actually flush the data out, issue WRITE_END to master, we should use the `mtime` from client rather than using the localtime in master. Does this sounds reasonable?

**Comment by acid-maker** [comment]
Sorry for very late answer, but we had a lot of other issues.

Full tracking of atime/mtime has been added in version 3.0.74. All chunk-related commands read_chunk,write_chunk and write_end change atime/mtime only if client version is lower than 3.0.74 (because old clients are unable to track atime/mtime, so old approach should be applied). For newer clients atime and mtime are updated independently using other commands. Are you sure that you are using version newer than 3.0.74?

I've just checked and it looks that untar works correctly on the newest version of MooseFS.
